ui/pages/confirmations/components/gas-timing/gas-timing.component.js

```diff
@@ -1,9 +1,18 @@
-import React, { useContext, useEffect, useState } from 'react';
 import BigNumber from 'bignumber.js';
```

Comment:
This component also has some tech debt, we can may be create a followup task to re-write it.